runtime/vm/ValueTypeHelpers.hpp

```diff
@@ -237,24 +237,48 @@ class VM_ValueTypeHelpers {
 		return acmpResult;
 	}
 
+	/*
+	* Determines if a name or a signature pointed by a J9UTF8 pointer is a Qtype.
+	*
+	* @param[in] utfWrapper J9UTF8 pointer that points to the name or the signature
+	*
+	* @return TRUE if the name or the signature pointed by the J9UTF8 pointer is a Qtype, FALSE otherwise
```

Comment:
Sorry to nitpick, but these ones should be lower case (and the other function below).